src/components/Todo.js

```diff
@@ -0,0 +1,55 @@
+import { useEffect, useState } from 'react';
+import PropTypes from 'prop-types';
+
+import remove from '../media/remove.png';
+
+// eslint-disable-next-line object-curly-newline
+const Todo = ({ id, description, completed, editItem, removeItem }) => {
+  const [edited, setEdited] = useState(() => ({ id, description, completed }));
+  const [item, setItem] = useState(true);
+
+  const handleSubmit = (e) => {
+    e.preventDefault();
+    setItem(true);
+  };
+
+  useEffect(() => {
+    editItem(edited);
+  }, [edited, edited.completed, edited.description, editItem]);
+
+  return (
+    <div>
+      {item ? (
+        <div onDoubleClick={() => setItem(false)} className="todo-item">
+          <input
```

Comment:
### Optional
I think you should use the 'onClick' event listener instead of the 'onDoubleClick' due to ease of functionality